site/content/en/docs/Installation/_index.md

```diff
@@ -22,9 +22,7 @@ description: >
 - Game Servers must have the [game server SDK]({{< ref "/docs/Guides/Client SDKs/_index.md"  >}}) integrated, to manage Game Server state, health checking, etc.
 
 {{< alert title="Warning" color="warning">}}
-Later versions of Kubernetes may work, but this project is tested against {{% k8s-version %}}, and is therefore the supported version.
-Agones will update its support to the n-1 version of what is available across the majority of major cloud providers - GKE, EKS and
-AKS, while also ensuring that all Cloud providers can support that version.
+This release has been tested against Kubernetes versions {{% k8s-version %}}. Other versions may work, but are unsupported.
```

Comment:
On this line, would it make sense to say "has been tested against Kubernetes versions ... on GKE." to make it clear? @markmandel